code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -33,6 +33,8 @@
 	mob_swap_flags = HUMAN|SIMPLE_ANIMAL|SLIME|MONKEY
 	mob_push_flags = ALLMOBS
 
+	meat_type = 0

```

Comment:
What if the code assumes `meat_type` is a type of meat, and runtimes as a result of this?